client/webserver/site/src/html/markets.tmpl

```diff
@@ -646,8 +653,8 @@
                   <td><img class="micro-icon" data-icon="to"> [[[Redeem]]]</td>
                 </tr>
                 <tr>
-                  <td><span id="vSwapFeesLow"></span> (<span id="vSwapFeesLowPct"></span>)</td>
-                  <td><span  id="vRedeemFeesLow"></span> (<span  id="vRedeemFeesLowPct"></span>)</td>
+                  <td><span id="vSwapFeesLow"></span> <span id="vSwapFeesLowPct"></span></td>
+                  <td><span  id="vRedeemFeesLow"></span> <span  id="vRedeemFeesLowPct"></span></td>
```

Comment:
If you are doing another commit, can you remove the weird extra space after `span` in this element tag?